This patch is created by Oracle to direct users to base64(1) as
we do not ship mmencode(1).

The ISC have previously been informed, it is an Oracle specific patch.

--- bind-9.18.9/doc/man/rndc.conf.5in
+++ bind-9.18.9/doc/man/rndc.conf.5in
@@ -86,8 +86,8 @@ authentication key. The base\-64 string
 .sp
 There are two common ways to generate the base\-64 string for the secret.
 The BIND 9 program \fI\%rndc\-confgen\fP can be used to generate a random
-key, or the \fBmmencode\fP program, also known as \fBmimencode\fP, can be
-used to generate a base\-64 string from known input. \fBmmencode\fP does
+key, or the \fBbase64\fP program, can be
+used to generate a base\-64 string from known input. \fBbase64\fP does
 not ship with BIND 9 but is available on many systems. See the Example
 section for sample command lines for each.
 .SH EXAMPLE
@@ -175,9 +175,9 @@ A complete \fBrndc.conf\fP file, includi
 is written to the standard output. Commented\-out \fBkey\fP and
 \fBcontrols\fP statements for \fI\%named.conf\fP are also printed.
 .sp
-To generate a base\-64 secret with \fBmmencode\fP:
+To generate a base\-64 secret with \fBbase64\fP:
 .sp
-\fBecho "known plaintext for a secret" | mmencode\fP
+\fBecho "known plaintext for a secret" | base64\fP
 .SH NAME SERVER CONFIGURATION
 .sp
 The name server must be configured to accept rndc connections and to
@@ -187,7 +187,7 @@ \fBcontrols\fP statement in the BIND 9 A
 details.
 .SH SEE ALSO
 .sp
-\fI\%rndc(8)\fP, \fI\%rndc\-confgen(8)\fP, \fBmmencode(1)\fP, BIND 9 Administrator Reference Manual.
+\fI\%rndc(8)\fP, \fI\%rndc\-confgen(8)\fP, \fBbase64(1)\fP, BIND 9 Administrator Reference Manual.
 .SH AUTHOR
 Internet Systems Consortium
 .SH COPYRIGHT
